core-services/prow/02_config/openshift/cloud-provider-powervs/_pluginconfig.yaml

```diff
@@ -3,6 +3,23 @@ approve:
   repos:
   - openshift/cloud-provider-powervs
   require_self_approval: true
+label:
+  restricted_labels:
+    openshift/cloud-provider-powervs:
+    - allowed_teams:
+      - openshift-staff-engineers
```

Comment:
do we need to add openshift-staff-engineers? I thought they already had the powers, perhaps by different means.